tests/test_issue_new_script.py

```
@@ -0,0 +1,292 @@
+        # ה-id של Issue חוסם הוא מזהה פנימי, ומכוון **אינו** שווה למספרו:
+        # שליחת המספר במקומו היא הבאג שהבדיקה מחפשת.
+        'case "$*" in\n'
+        '  *"--method POST"*) exit 0 ;;\n'
+        '  *"dependencies/blocked_by"*) echo "${FAKE_DEPS:-1}"; exit 0 ;;\n'
+        '  *"--jq .id"*) echo 3300771234; exit 0 ;;\n'
+        "esac\n"
+        "exit 0\n",
+        encoding="utf-8",
+        newline="\n",
+    )
+    os.chmod(bindir / "gh", 0o755)
```

Ruff (0.16.3)

[error] `os.chmod` setting a permissive mask `0o755` on file or directory

(S103)